rewrite-java-8/src/main/java/org/openrewrite/java/ReloadableJava8JavadocVisitor.java

```diff
@@ -621,11 +621,14 @@ public Tree visitParam(ParamTree node, List<Javadoc> body) {
                         methodRefType
                 );
             } else {
+                if (qualifier == null) {
+                    return new J.Identifier(randomId(), Space.EMPTY, Markers.EMPTY, emptyList(), name.getSimpleName(), qualifierType, fieldRefType);
+                }
                 return new J.MemberReference(
                         randomId(),
-                        qualifier == null ? Space.EMPTY : qualifier.getPrefix(),
+                        qualifier.getPrefix(),
                         Markers.EMPTY,
-                        JRightPadded.build(qualifier == null ? null : qualifier.withPrefix(Space.EMPTY)),
+                        JRightPadded.build(qualifier.withPrefix(Space.EMPTY)),
```

Comment:
We'd considered returning an empty identifier here but this already happens for example with this piece of JavaDoc:
```
/**
 * {@link #n} refers to Inner
 */
```
so there would be no trivial way to determine whether to place the `#` or not, both are valid JavaDoc